sales.html

```diff
@@ -5,6 +5,24 @@
   </head>
   <body>
     <main>
+      <h2>Add a store</h2>
+      <form id="addStoreForm">
+        <fieldset>
+          <legend>About the store</legend>
+          <label>Name</label>
+          <input name="name" type="text" />
+        </fieldset>
+        <fieldset>
+          <legend>Store statistics</legend>
+          <label>Minimum customers per hour</label>
+          <input name="minHourlyCustomer" type="text" />
+          <label>Maximum customers per hour</label>
+          <input name="maxHourlyCustomer" type="text" />
+          <label>Average cookies per customer</label>
+          <input name="hourlyCookiesPerCustomer" type="text" />
```

Comment:
You should add form validation here to make sure nothing except a number is entered: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input